app/helpers/application_helper.rb

```diff
@@ -1,2 +1,5 @@
 module ApplicationHelper
+  def flash_key(key)
+    %w(info warning danger success default).include?(key.to_s) ? key : 'info'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.